src/WorkflowCore/Services/FluentBuilders/StepBuilder.cs

```diff
@@ -1,13 +1,16 @@
 ﻿using System;
 using System.Collections;
+using System.Collections.Generic;
 using System.Linq.Expressions;
 using WorkflowCore.Interface;
 using WorkflowCore.Models;
 using WorkflowCore.Primitives;
 
 namespace WorkflowCore.Services
 {
-    public class StepBuilder<TData, TStepBody> : IStepBuilder<TData, TStepBody>, IContainerStepBuilder<TData, TStepBody, TStepBody>
+    public class StepBuilder<TData, TStepBody> 
+        : IContainerStepBuilder<TData, TStepBody, TStepBody>,
+          ICatchStepBuilder<TData, TStepBody>
```

Comment:
Not sure I understand why the class would implement ICatchStepBuilder?

Reply:
Because we need to return `ICatchStepBuilder` after `Catch()`.
```
    public class StepBuilder<TData, TStepBody> 
        : IContainerStepBuilder<TData, TStepBody, TStepBody>,
          ICatchStepBuilder<TData, TStepBody>,
          IStepBuilder<TData, TStepBody>, 
          ITryStepBuilder<TData, TStepBody>
```
This would be more verbose, but two last lines are redundant in that case.